examples/maps_heatmaps/src/queryDB.ts

```diff
@@ -0,0 +1,53 @@
+import { database } from "./index";
+import * as firebase from "firebase";
+import * as geofirestore from "geofirestore";
+
+/* returns the filtered collection by the different queries*/
+function getQuiredCollection(
+  center: firebase.firestore.GeoPoint,
+  radius: number,
+  labels: string[],
+  year?: number,
+  month?: number,
+  day?: number
+): geofirestore.GeoQuery {
+  const GeoFirestore = geofirestore.initializeApp(database);
+  const geoCollection = GeoFirestore.collection("images");
+  let dataRef: geofirestore.GeoQuery = geoCollection
+    .near({
+      center: center,
+      radius: radius,
+    })
+    .where("labels", "array-contains-any", labels);
+  if (year != undefined) dataRef = dataRef.where("year", "==", year);
+  if (year != undefined && month != undefined)
+    dataRef = dataRef.where("month", "==", month);
+  if (year != undefined && month != undefined && day != undefined)
+    dataRef = dataRef.where("day", "==", day);
+  return dataRef;
+}
+
+/* displays the relevant images on the map
+given the filtered collection and the heapmap*/
+function getPointsFromDB(
```

Comment:
rename to better explain what it does. It queries and displays the points. maybe `updateHeatmapFromQuery`?
A function called `getSomething` suggests that it returns data when called, which is not the case here.

Reply:
renamed to updateHeatmapFromQuery:-)